### DISCUSSION TITLE
Update catalog item via patchListingsItem

### DISCUSSION THREAD

**Discussion by BarnetMichal** [amzn/selling-partner-api-models#4071] [Q&A] [answered]
I would like to edit the bullet_point of a product that has already been added to the catalog. It was added via POST_PRODUCT_DATA. I would like to edit it via patchListingsItem.  When I try this I get the error '"The field 'value' for the attribute 'brand' does not have enough values. The required minimum is '1' value(s)."' In the Product schema, brand/value is marked as "editable": false. So do I have to first create a listing via putListingsItem even if the product already exists in the catalog and then call patchListingsItem on it?

**Comment by chapmanjw** [comment, marked as answer]
@BarnetMichal What does your PATCH request look like? (Can you paste a copy of the payload?) Typically, you would only see errors related to the `brand` attribute on a PATCH request if you provided an empty `brand` attribute as part of the PATCH request. A request like the following should work (replacing values with your own, such as product type, language, and values):

```
{
  "productType":"LUGGAGE",
  "patches":[
    {
      "op":"replace",
      "path":"/attributes/bullet_point",
      "value":[
        {
          "value":"Bullet Point 1",
          "language_tag":"en_US"
        },
        {
          "value":"Bullet Point 2",
          "language_tag":"en_US"
        }
      ]
    }
  ]
}
```

**Reply by BarnetMichal** [reply]
Yes, I only sent the brand attribute with language_tag and market_id. Omitting the brand attribute helped, thanks!
